### DISCUSSION TITLE
Are there any plans for a Welsh language model?

### DISCUSSION THREAD

**Discussion by ghost** [explosion/spaCy#12768] [Help: Other Questions] [answered]
Hi, I was wondering if there are any plans in the works to publish models that support the Welsh language?

**Comment by danieldk** [comment, marked as answer]
We currently don't have any Welsh models planned. There is a small [UD treebank](https://universaldependencies.org) for Welsh that you could try to train a model on. If you want to make a model for Welsh, you could also look at the [Adding models for new languages master thread](https://github.com/explosion/spaCy/discussions/3056) and the [language support discussions](https://github.com/explosion/spaCy/discussions/categories/language-support).
